src/main/java/pl/knpj/servlet/model/Answer.java

```diff
@@ -0,0 +1,42 @@
+package pl.knpj.servlet.model;
+
+
+import java.util.Collection;
+
+/**
+ * Created by vadikms on 29.11.16.
+ */
+public class Answer {
```

Comment:
Jest to jakieś rozwiązanie, ale chcesz żeby była to oddzielna klasa dla każdego POJO?